### ISSUE TITLE
Add an Exercism.org track for nu programming language

### ISSUE THREAD

**Issue by luccahuguet** [nushell/nushell#6699]
### Describe the solution you'd like

Add an Exercism.org track for the nu programming language! 😍 

It would be very helpful for those of us new to nu. 

Even 3 exercises could be very effective!

**Comment by jacek-kurlit** [comment]
Hi,
Usually I learn new language on exercism. It helps a lot during learning to solve some problems and see other community members solutions
 is there any update on this one?

**Comment by luccahuguet** [comment]
> Hi, Usually I learn new language on exercism. It helps a lot during learning to solve some problems and see other community members solutions is there any update on this one?

did you run `tutor` on nushell? not the same thing, but definitely helps

**Comment by jacek-kurlit** [comment]
I will take a look thank you!
My question still remains though, are there any plans to create nu track on exercism?

**Comment by AucaCoyan** [comment]
I started yesterday with a few tracks and I'm impressed on how good the quality of the content is.
One question though: the API of nushell is not yet stable. It's not on the alpha stage, or somethings won't change soon, but doing this course would need some maintenance over time (even just a little).
Because it's a OSS organization, we are short on hands for everything. Is it too much/too hard of a task asking you to complete it @jacek-kurlit ?

**Comment by jacek-kurlit** [comment]
I have never created and maintain a track on exercism but I can give it a try.
Although I have my concerns: I'm at the beginning of learning nu and exercism usually expects some "idiomatic solution" so probably I'm not the  best candidate here.

**Comment by AucaCoyan** [comment]
Nah, don't worry, it's always a group effort. An idiom is probably what more people write out, not because "it's a correct way of writing nushell" like an english grammar would.
We can start the ball rolling and see where it goes 🙌🏼

**Comment by jacek-kurlit** [comment]
Ok, I will try to gather some time to set up track for nu but I cannot promise anything as I have my own open source side project

**Comment by cboecking** [comment]
Team, I have been working toward learning the 'nushell way of thinking' for quite some time. I do not know how, but I did not know that `tutor` existed. 

I believe `tutor` should be one of the first items mentioned in the getting started section of the book. I believe it should be mentioned multiple times in different parts of the getting started (namely: thinking in nu).

This is a great tool, and I believe if any effort is applied to learning nu, it should be in this tool!

The only other way to make it better is to make the nushell docs interactive per this discussion - https://discord.com/channels/601130461678272522/1326236285211381840/1327406898042896385

Thank you for your time and consideration!!

**Comment by 0x4D5352** [comment]
Could I be assigned this issue? I was already working on a Nushell doc for [Learn X in Y Minutes](https://learnxinyminutes.com) and would like to combine a bunch of educational updates into one larger project. Would also involve improvements to `tutor` as @cboecking mentioned.

**Comment by AucaCoyan** [comment]
I'm not a member of the nushell org, but I think nobody is doing this at the moment.
You can start fresh!

**Comment by luccahuguet** [comment]
> Could I be assigned this issue? I was already working on a Nushell doc for [Learn X in Y Minutes](https://learnxinyminutes.com) and would like to combine a bunch of educational updates into one larger project. Would also involve improvements to `tutor` as [@cboecking](https://github.com/cboecking) mentioned.

you can just do things 👍🏼

**Comment by 0x4D5352** [comment]
fair! just saw @jacek-kurlit mention it and wanted to check
